src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Emitters/EndpointParameterEmitter.cs

```diff
@@ -243,7 +243,7 @@ internal static void EmitBindAsyncPreparation(this EndpointParameter endpointPar
                 codeWriter.WriteLine($"var {endpointParameter.EmitTempArgument()} = await {unwrappedTypeString}.BindAsync(httpContext);");
                 break;
             default:
-                throw new Exception("Unreachable!");
+                throw new InvalidOperationException("Unreachable!");
```

Comment:
Would `UnreachableException` make more sense here (and below)?

```suggestion
                throw new UnreachableException();
```

Reply:
`UnreachableException` is a good choice here. We really don't expect it to be hit although our coverage isn't 100% right now for the RDG work we are doing so it could happen.

Reply:
Hidden in the commit message:
> UnreachableException not available in .NET Standard

Reply:
Yep. I would have used `UnreachableException` if it was available. I think leaving it as `Exception` is fine considering it should be unreachable. If we just want to avoid it showing up when people search the codebase for "new Exception", `InvalidOperationException` is fine too.

Reply:
My recommendation would be to work towards avoiding CA2201 warnings entirely. If it's meant to be unreachable, there's a more appropriate exception than `System.Exception`.

Reply:
Switched to `NotImplementedException` with more context.